platform-coop-toolkit/coop-library
# fix: remove favorite from page after deletion (fix #220)

## Description

Fixes #220.

## Steps to test

1. Add some favorites.
2. Visit favorites page.
3. Delete a favorite.

**Expected behavior:** Favorite card is removed.

## Additional information

Not applicable.

## Related issues

- Resolves #220.
